DESCRIPTION

```diff
@@ -39,6 +38,8 @@ Imports:
     base64enc,
     shiny,
     rstudioapi
+Remotes:
+    dplyr (>= 0.5.0.9000)
```

Comment:
You want this to be e.g. `hadley/dplyr`; that signals that `dplyr` should be installed using `devtools::install_github("hadley/dplyr")`.